k8s/gke/k8s-installer/create-gke-cluster.yml

```diff
@@ -19,6 +19,17 @@
             cluster_name: "{{ cluster_name.stdout }}"
           when: cluster_name.stdout is defined
 
+        - name: Generate SSH Keys 
+          shell: yes y | ssh-keygen -t rsa -N "" -f $HOME/.ssh/openebs_rsa -C openebs
```

Comment:
File name provided for storing ssh key, it should be generic or related to litmus